src/main/java/org/opentripplanner/middleware/triptracker/TripInstruction.java

```diff
@@ -0,0 +1,10 @@
+package org.opentripplanner.middleware.triptracker;
+
+public enum TripInstruction {
+
```

Comment:
Add a comment saying that more instruction types can be added.

Reply:
Addressed in https://github.com/ibi-group/otp-middleware/pull/203/commits/ec0152118d75a99b1cdf92e774c3e088927e6273